projects/equation/index.js

```diff
@@ -0,0 +1,39 @@
+const { sumTokens2 } = require("../helper/unwrapLPs");
+const { getLogs } = require('../helper/cache/getLogs')
+
+
+const config = {
+    arbitrum: { factory: '0xA91680161fBCeA942e164B42445aD6130D01541F', stakingContract: '0x3C77EEB8eC4716a6389a522eD590FbbD261ABE8e', EQU: '0x87AAfFdF26c6885f6010219208D5B161ec7609c0',V3Pool:['0xaFF6b531F5a06c00A21e34FD7BCe9cf024ef45F0','0xc7f83c20ad7DA85661546FECC19C27cb22ED53E3','0x7671e24bBD8B1FD52A4d936C8170927Fa36BAE3A','0xC0cF0f380DdB44dBCAF19A86d094C8BBA3efa04a'] },
```

Comment:
hi @DIO1009 , thanks for the PR, the code is mostly good except for the pool2 part, you can only count the v3 nfts locked in your smart contract towards your pools, also unsure what last 3 addresses are